src/garage/tf/policies/gaussian_mlp_task_embedding_policy.py

```diff
@@ -237,8 +238,8 @@ def get_action(self, observation):
                     A is the dimension of action.
 
         """
-        obs, task = self.split_augmented_observation(observation)
-        return self.get_action_given_task(obs, task)
+        actions, agent_infos = self.get_actions([observation])
```

Comment:
wait, is this right?

Reply:
is there something about it that would be wrong for task embedding policies? It passes all of the current tests. @naeioi could you chime in here?

Reply:
`self.get_action_given_task(obs, task)` is a pretty essential part of task embeddings...

Reply:
I see

Reply:
I reverted this change and things are back to their original behavior.